### PR TITLE
Add style.puml and Architecture Diagram (work in progress)

### PR DESCRIPTION
**Pull request by ngdeqi**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -299,9 +299,9 @@ When client rents a property, the client and property form a pair.
 
 #### Pair
 
-Here is a sequence diagram for the pair command, called from ```Duke.java```:
+The sequence diagram for the pair command is called is shown below:
```
Comment (wilsonngja) [inline docs/DeveloperGuide.md:302]: I see that you've change the phrasing to make it more formal. Great work.

File: docs/diagrams/ArchitectureDiagram.puml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+ClientList -[CLIENT_LIST_COLOR]-> Client
+
+PairingList .[PAIRING_LIST_COLOR].> Client
+PairingList .[PAIRING_LIST_COLOR].> Property
+
+Command .[COMMAND_COLOR].-> Storage
+Command .[COMMAND_COLOR].-> UI
+
+
+@enduml
```
Comment (wilsonngja) [inline docs/diagrams/ArchitectureDiagram.puml:54]: I really like how you started using PlantUML. It's really neat and tidy.

**Review by wilsonngja — COMMENTED** [review]
Good work and it LGTM.

**Review by FeliciaBeatrice — APPROVED** [review]
everything looks good! LGTM

**Pull request merged**